webpack/JobWizard/steps/HostsAndInputs/SelectGQL.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { SelectVariant } from '@patternfly/react-core';
+import {
+  useForemanOrganization,
+  useForemanLocation,
+} from 'foremanReact/Root/Context/ForemanContext';
+import { HOSTS, HOST_GROUPS, dataName } from '../../JobWizardConstants';
+import { SearchSelect } from '../form/SearchSelect';
+
+export const queries = {
+  [HOSTS]: gql`
+    query($search: String!) {
+      hosts(first: 100, last: 100, search: $search) {
+        totalCount
+        edges {
+          node {
+            name
+          }
+        }
+      }
+    }
+  `,
+  [HOST_GROUPS]: gql`
+    query($search: String!) {
+      hostgroups(first: 100, last: 100, search: $search) {
+        totalCount
+        edges {
```

Comment:
Since you do not request any attributes on edges, it is possible to have:

```
totalCount
nodes {
  name
}
`
It might help with mapping on l.60

Reply:
Thanks, changed!